views/index.ejs

```diff
@@ -5,88 +5,26 @@
-                <img class="block h-8 w-auto lg:hidden" src="https://tailwindui.com/img/logos/mark.svg?color=indigo&shade=500" alt="Your Company">
-                <img class="hidden h-8 w-auto lg:block" src="https://tailwindui.com/img/logos/mark.svg?color=indigo&shade=500" alt="Your Company">
-              </div>
-              <div class="hidden sm:ml-6 sm:block">
-
-                <div class="flex space-x-4">
-                  <!-- Current: "bg-gray-900 text-white", Default: "text-gray-300 hover:bg-gray-700 hover:text-white" -->
-                  <% if (!user) { %>
-                    <a href="/login" class="hover:bg-gray-700 text-gray-300 hover:text-white rounded-md px-3 py-2 text-sm font-medium"
-                      aria-current="page">Login</a>
-                    <a href="/signup"
-                      class="text-gray-300 hover:bg-gray-700 hover:text-white focus:bg-gray-900 rounded-md px-3 py-2 text-sm font-medium">Signup</a>
-                    <% } else {%>
-                      <a href="/questions" class="hover:bg-gray-700 text-white rounded-md px-3 py-2 text-sm font-medium"
-                      aria-current="page">Questions</a>
-                      <a href="/logout"
-                      class="text-gray-300 hover:bg-gray-700 hover:text-white focus:bg-gray-900 rounded-md px-3 py-2 text-sm font-medium">Logout</a>
-                    <% } %>
-                </div>
-              </div>
-            </div>
-
-            </div>
-          </div>
-        </div>
-
-        <!-- Mobile menu, show/hide based on menu state. -->
-        <div class="sm:hidden" id="mobile-menu">
-          <div class="space-y-1 px-2 pb-3 pt-2">
-            <!-- Current: "bg-gray-900 text-white", Default: "text-gray-300 hover:bg-gray-700 hover:text-white" -->
-            <a href="#" class="bg-gray-900 text-white block rounded-md px-3 py-2 text-base font-medium" aria-current="page">Dashboard</a>
-            <a href="#" class="text-gray-300 hover:bg-gray-700 hover:text-white block rounded-md px-3 py-2 text-base font-medium">Team</a>
-            <a href="#" class="text-gray-300 hover:bg-gray-700 hover:text-white block rounded-md px-3 py-2 text-base font-medium">Projects</a>
-            <a href="#" class="text-gray-300 hover:bg-gray-700 hover:text-white block rounded-md px-3 py-2 text-base font-medium">Calendar</a>
-          </div>
-        </div>
-      </nav>
-
-<div class="flex flex-col items-center justify-center h-[calc(100%-4rem)] py-2 overflow-hidden">
-  <div class="flex flex-col items-center justify-center  w-screen">
-    <h1 class="text-6xl font-bold text-white text-center">
-      Would You Rather
-    </h1>
-    <h2 class="text-2xl text-gray-100 text-center">
-      Have Fun With Your Friends And Uncover Personal Insight
-    </h2>
-    <%if (!user) { %>
-    <a href="/signup" class="mt-8 px-6 py-3 text-white font-semibold rounded-lg bg-green-500 hover:bg-green-600">Get
-      Started</a>
-    <% } %>
```

Comment:
as this navbar is literally the same as the testNavbar, imported the testNavbar instead